{
  "dataservicehandler.update.status.success": "Daten wurden gesendet und geladen.",
  "dataservicehandler.update.status.authentication_failed": "Autentifizierung fehlgeschlagen, überprüfe API Schlüssel.",
  "dataservicehandler.update.status.no_such_user": "Kein solcher Benutzer im Remote-Service definiert.",
  "dataservicehandler.update.status.no_auth_key": "Es wurde kein Autorisierungsschlüssel konfiguriert.",
  "dataservicehandler.update.status.launch_fail": "Die Remote API konnte nicht gestartet werden.",
  "dataservicehandler.update.status.rate_limit": "Fehler beim Aktualisieren aufgrund von Geschwindigkeitsbeschränkungen.",
  "dataservicehandler.delete.status.success": "Alle Account- und PhantomBotDE-Daten wurden vom Remote-Server entfernt.",
  "dataservicehandler.delete.status.authentication_failed": "Authentifizierung fehlgeschlagen, API-Schlüssel prüfen.",
  "dataservicehandler.delete.status.no_such_user": "Kein solcher Benutzer im Remote-Service definiert.",
  "dataservicehandler.delete.status.no_auth_key": "Es wurde kein Autorisierungsschlüssel konfiguriert.",
  "dataservicehandler.delete.status.launch_fail": "Die Remote-API konnte nicht gestartet werden."
}